src/components/Subscribe/Subscribe.tsx

```diff
@@ -33,6 +33,22 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
    * Subscribe title text
    */
   title?: string;
+  /**
+   * Subscribe loading text
+   */
+  loadingText?: string;
+  /**
+   * Subscribe error text
+   */
+  invalidText?: string;
+  /**
+   * Subscribe state for loading or error
+   */
+  subscriptionState?: 'loading' | 'invalid' | null;
+  /**
+   * Subscribe input change handler
+   */
+  onInputChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
rather than track every change event to the `<input`, I think it would be better to use an `onSubscribe` action that only triggers when the form submission is successful, and passes the current state of the input